### ISSUE TITLE
Add skos:Concept to the domain and range of ceterms:precedes

### ISSUE THREAD

**Issue by stuartasutton** [CredentialEngine/Schema-Development#809]
In order to indicate sequence in a Progression Model, add [skos:Concept](http://www.w3.org/2004/02/skos/core#Concept) to the domain and range of [ceterms:precedes](https://purl.org/ctdl/terms/precedes). 

Regarding [ceterms:precedes](https://purl.org/ctdl/terms/precedes) (and [ceterms:prerequisite](https://purl.org/ctdl/terms/prerequisite) in this context), see #807.

**Comment by siuc-nate** [comment]
I think this might be a job for ceasn:listID instead, since that would also handle multi-layered concept schemes more cleanly. For example:
```
1. Thing A
  1.a. Thing B
  1.b. Thing C
  1.c. Thing D
    1.c.1. Thing E
    1.c.2. Thing F
2. Thing G
```
The listID handles nesting cleanly (and may already be included in the source data), rather than asking the user to figure out "Should `Thing A` or `Thing D` or `Thing F` be marked as preceding `Thing G`?"

**Comment by stuartasutton** [comment]
Nate, [ceasn:listID](https://purl.org/ctdlasn/terms/listID) is intended to capture express notations found in the source data. Sequencing is progression model data is most frequently implied--e.g., "1st Term", "2nd Term" etc. The value of a [ceasn:listID](https://purl.org/ctdlasn/terms/listID) is frequently and best displayed.  We already have [ceterms:precedes](https://purl.org/ctdl/terms/precedes), which fits the context perfectly.

**Comment by philbarker** [comment]
@stuartasutton in the [N Dakota Cyber model](https://app.diagrams.net/#G1yeUJ5hPnwfc9COWQVQWWPd18FVwuYdy6), I used `ProgressionLevel`s to indicate semester I, II, III & IV, and indicated a need to indicate that `ProgressLevel` Semester I precedes `ProgressionLevel` Semester II and so on: i.e. adding [ceasn:ProgressionLevel](http://purl.org/ASN/schema/core/ProgressionLevel) to domain and range of [ceterms:precedes](https://purl.org/ctdl/terms/precedes). 

This sounds like the same use case, so do you have a different solution in mind or am I missing a link between skos:Concept and ceasn:ProgressionLevel?

Related: how do you say that a `ProgressionLevel` is in a `ProgressionModel` ?

**Comment by stuartasutton** [comment]
Phil, this is in my growing list of Pathway changes at https://github.com/CredentialEngine/Schema-Development/issues/807#issuecomment-1004378147

**Comment by stuartasutton** [comment]
Is issue is resolved in #807.
